deploy/helm-chart/README.md

```diff
@@ -69,6 +69,16 @@ helm install my-release oci://ghcr.io/nginxinc/charts/nginx-gateway-fabric  --cr
 This will install the latest stable release. To install the latest version from the `main` branch, specify the
 `--version 0.0.0-edge` flag when installing.
 
+> **Note**
```

Comment:
I don't think we should include details on `kind` deployments in our installation docs. `kind` is designed for development or CI purposes, not for production. I think this note would be better in the `running on kind` doc. 

However, I agree with @kevin85421 that this is a bad user experience for someone trying out NGF on `kind`, and I think we should address that.

It seems like there are two problems with this install command:
1. The `--wait` option causes installation to time out if you are installing on a cluster that does not support service of type LoadBalancer
2. There's no way of knowing that this command installs a service of type LoadBalancer without looking at all the default values.

For 1, do we need to include the `--wait` option in our default install command? Could we add a note to the helm output that tells users the installation takes a minute and to use `helm status` to check the status of the release?

For 2, I think we should add text that explains the command creates a service of type LoadBalancer and provide other examples of how to install with a different service type and with no service. Service is important enough to users that I think having multiple installation examples is warranted.

Reply:
Good callouts. Maybe we could add a note before the install command to say something like "By default a LoadBalancer type service is deployed when running the following command. If you don't want a service add this flag...., or want Nodeport add this flag...."

I personally like having the `--wait` command be explicit because it ensures things are running before a user or robot proceeds with creating resources.

Reply:
> . Maybe we could add a note before the install command to say something like "By default a LoadBalancer type service is deployed when running the following command. If you don't want a service add this flag...., or want Nodeport add this flag...."

That works. I also  like the idea of providing full install examples for each option:
```
helm install ... --service.create=false
```
so that it is easy for the user to copy and paste the command they want to use.

> I personally like having the --wait command be explicit because it ensures things are running before a user or robot proceeds with creating resources.

That's fair. Can we provide it as an option instead of the default command?

Reply:
How best should we organize this? We're gonna end up with a lot of command examples.

Reply:
@kevin85421 Would you mind moving this note to the `running-on-kind.md` doc? We'll handle adding similar information to the installation doc in a separate. PR, per the discussion here.

Reply:
Thanks, @kate-osborn and @sjberman! I have already moved the note to `running-on-kind.md`.